packages/components/inputs/async-select-input/src/async-select-input.stories.tsx

```diff
@@ -144,3 +144,35 @@ BasicExample.args = {
   showOptionGroupDivider: false,
   iconLeft: undefined,
 };
+
```

Comment:
This is mainly to testing simplicity and will be removed before merging.

Reply:
For some reason, label/placeholder and input break lines after selection:

![image](https://github.com/user-attachments/assets/4e57f781-85b4-47ee-8fe1-87872ecad829)
![image](https://github.com/user-attachments/assets/277b8d1c-f1de-4c9d-9752-b1ef53952530)